src/Containers/VendorsList/VendorsList.stories.tsx

```diff
@@ -0,0 +1,247 @@
+import React from 'react';
+import { Story, Meta } from '@storybook/react';
+import { TableView } from '@styled-icons/material/TableView';
+import { List } from '@styled-icons/bootstrap/List';
+import { ViewGrid } from '@styled-icons/heroicons-solid/ViewGrid';
+import { Profile } from './Profile'; 
+import { TagContainer } from './TagContainer';
+import { SmallText } from '../../Text/SmallText';
+import { VendorsList, IVendorsListProps } from './VendorsList';
+import { createStoryTitle } from '../../Constants';
+
+export default {
+    title: createStoryTitle('Vendors List'),
+    component: VendorsList,
+} as Meta;
+
+const isHoverable = false;
+    
+const getVendorsListProps = (): IVendorsListProps => ({
+    filterButtonText: 'Apply',
+    filterTitleText: 'Filters',
+    vendorsHeaderText: 'Clients',
+    rightButtonText: 'Add Client',
+    navigationBarItems: [
+        {
+            icon: TableView,
+            label: 'Overview',
+        },
+        {
+            icon: List,
+            label: 'List View',
+        },
+        {
+            icon: ViewGrid,
+            label: 'Segment',
+        },
+    ],    
+    filterItems:  [
+        {
+            title: 'Email',
+            selectOptions: ['Contains', 'Equals'],
+            placeholder: 'Add email'
+        },
+        {
+            title: 'First Name',
+            selectOptions: ['Starts with', 'Equals'],
+            placeholder: 'Add first name'
+        },
+        {
+            title: 'Last Name',
+            selectOptions: ['Starts with', 'Equals'],
+            placeholder: 'Add last name'
+        },
+        {
+            title: 'Tag',
+            selectOptions: ['Contains', 'Equals'],
+            placeholder: 'Add tag'
+        },
+        {
+            title: 'Created',
+            selectOptions: ['Created earlier than', 'Created on', 'Created later than'],
+            placeholder: 'Select date'
+        }
+    ],
+    columns: [
+        {
+            Header: 'Client',
+            Cell: (cell: any) => (
+                <Profile
+                    key={cell.row.original.id}
+                    name={cell.row.original.name}
+                    email={cell.row.original.email}
+                    imageUrl={cell.row.original?.imageUrl}
+                    profileProps={{ style: { marginTop: 'tpx' } }}
+                />
+            ),
+        },
+        {
+            Header: 'Tags',
+            Cell: (cell: any) => (
+                <TagContainer 
+                    tags={cell.row.original.tags} 
+                    isHoverable={isHoverable} 
```

Comment:
replace variable with false, unessesary abstraction. I can see false relates to isHoverable in this case becuase of the context